README.md

```diff
@@ -400,6 +400,12 @@ Validates MAC addresses.
 Examples:
 - `mac()`: Allows any valid MAC address
 
+### SemVer (Semantic Versioning) - `SemVer()`
+Validates [Semantic Versioning](https://semver.org/) strings.
+
+Examples:
+- `SemVer()`: Allows any valid SemVer string
```

Comment:
I think this should be `semver()` instead of `SemVer()` right?

Reply:
Looking at the rest of the values, for consistency, yes. Updated.

Although, intuitively, I'd have expected the UpperCamelCase version to be the correct one, to mirror the class definition. I suppose it uses the value of `tag` then.